test/e2e/tests/test_ca_activation.py

```diff
@@ -141,14 +114,18 @@ def test_ca_activation(self, acmpca_client, create_secret, test_create_ca):
 
         # Create k8s resource
         ref = k8s.create_reference(
-            CRD_GROUP, CRD_VERSION, RESOURCE_PLURAL,
+            CRD_GROUP, CRD_VERSION, "certificates",
             cert_name, namespace="default",
```

Comment:
The logic that is create/deleting a certificate should also be exported to a fixture function